Format all headers under ATen/cpu/vec, not just top-level

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #152388
* #152366
* #152365
* __->__ #152364

not formatting these seems like an oversight. Had to add a few clang-format suppressions to keep includes in the same order to avoid breaking builds.

This PR was generated using `lintrunner --paths-cmd "rg --files -g '*.h' aten/src/ATen/cpu/vec/" format`

Differential Revision: [D73802128](https://our.internmc.facebook.com/intern/diff/D73802128/)

cc @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10 @jerryzh168 @albanD

swolchok committed:
Format all headers under ATen/cpu/vec, not just top-level

not formatting these seems like an oversight. Had to add a few clang-format suppressions to keep includes in the same order to avoid breaking builds.

Differential Revision: [D73802128](https://our.internmc.facebook.com/intern/diff/D73802128/)

[ghstack-poisoned]

swolchok commented:
@albanD / @seemethere I'm getting yelled at by pr-sanity-checks, which says this PR is too long and I should contact you

malfet reviewed (approved):
Assuming it's a change generated by the tool, LGTM

swolchok commented:
> Assuming it's a change generated by the tool, LGTM

it's generated except for the two small uses of clang-format off/on

albanD commented:
Please share the command used to generate the diff so we can skip the pr-sanity-check !

swolchok commented:
> Please share the command used to generate the diff so we can skip the pr-sanity-check !

lintrunner -a

swolchok committed:
Update on "Format all headers under ATen/cpu/vec, not just top-level"


not formatting these seems like an oversight. Had to add a few clang-format suppressions to keep includes in the same order to avoid breaking builds.

Differential Revision: [D73802128](https://our.internmc.facebook.com/intern/diff/D73802128/)

cc jgong5 mingfeima XiaobingSuper sanchitintel ashokei jingxu10 jerryzh168

[ghstack-poisoned]

swolchok committed:
Update on "Format all headers under ATen/cpu/vec, not just top-level"


not formatting these seems like an oversight. Had to add a few clang-format suppressions to keep includes in the same order to avoid breaking builds.

Differential Revision: [D73802128](https://our.internmc.facebook.com/intern/diff/D73802128/)

cc jgong5 mingfeima XiaobingSuper sanchitintel ashokei jingxu10 jerryzh168

[ghstack-poisoned]

swolchok commented:
@albanD can you please advise on the process for bypassing pr-sanity-checks for this PR? (I understand I will need to rebase, but I don't know how we expect to proceed after that)

albanD commented:
Added the repro command to the PR description and appropriate label. Should be all good now !

swolchok committed:
Update on "Format all headers under ATen/cpu/vec, not just top-level"


not formatting these seems like an oversight. Had to add a few clang-format suppressions to keep includes in the same order to avoid breaking builds.

This PR was generated using `lintrunner --paths-cmd "rg --files -g '*.h' aten/src/ATen/cpu/vec/" format`

Differential Revision: [D73802128](https://our.internmc.facebook.com/intern/diff/D73802128/)

cc jgong5 mingfeima XiaobingSuper sanchitintel ashokei jingxu10 jerryzh168 albanD

[ghstack-poisoned]

swolchok commented:
@pytorchbot merge -i
